packages/core/src/routes/admin-user.ts

```diff
@@ -153,10 +153,10 @@ export default function adminUserRoutes<T extends AuthedRouter>(
     koaGuard({
       params: object({ userId: string() }),
       body: object({
-        username: string().regex(usernameRegEx).or(literal('')).nullable(),
+        username: string().nullable(),
```

Comment:
need to discuss whether allow spaces in the `username`

Reply:
Yep, lets do it tomorrow